less/index.less

```diff
@@ -49,11 +51,134 @@ table {
 }
 
 /* Set every element's box-sizing to border-box */
+
+// Variables
+
+@text-color: @firebrick;
+@firebrick: #B22222;
+@main-font: Georgia;
+
+
+// Mixins
+.bottom-content () {
+	
+	display: flex;
+	justify-content: center;
+
+}
+
+.intro-content () {
+	font-size: 2rem;
+}
+// General Styles
+
 * {
     box-sizing: border-box;
 }
 
+html {
+	
+	font-family: @main-font;
+  }
+
 html, body {
-    height: 100%;
+	height: 100%;
+	max-width: 100%;
+	@media (max-width: 500px) {
+		max-width: 100%;
+		background-color: pink;
+		display: flex;
+		flex-direction: column;
+		}
+
+}
+
+.intro-content img {
+	max-width: 100%;
+	height: auto;
 }
 
+.container {
```

Comment:
all of the following styles should have been nested in the container as josh did.